jsx/Form.js

```diff
@@ -472,13 +477,17 @@ class SelectElement extends Component {
     // Default to empty string for regular select and to empty array for 'multiple' select
     const value = this.props.value || (multiple ? [] : '');
 
+    const label = this.props.label && (
+      <label className="control-label" htmlFor={this.props.id}>
+        {this.props.label}
+        {requiredHTML}
+      </label>
+    );
+
     return (
       <div className={elementClass}>
-        <label className="col-sm-3 control-label" htmlFor={this.props.label}>
-          {this.props.label}
-          {requiredHTML}
-        </label>
-        <div className="col-sm-9">
+        {label}
+        <div>
```

Comment:
After investigating, most of this actually seems to be on 24.1 now. The main different being that the value passed to `htmlFor` is `this.props.label` instead of `this.props.id`.

**to investigate**